### PR TITLE
REF: Move to params for better predictability

### PR DESCRIPTION
**Pull request by marcosrdz**
## Release Notes

- **New Features**
	- Enhanced `AddressInput` component to allow users to import files and choose photos directly.
	- Added new parameters in `SendDetails` for improved transaction details: `feeUnit`, `amountUnit`, `frozenBalance`, `transactionMemo`, `utxos`, `payjoinUrl`, and `isTransactionReplaceable`.
- **Bug Fixes**
	- Improved error handling for file and image selection in `AddressInput`.
- **Style**
	- Updated styling in `CoinControl` to improve layout consistency with increased modal height.
- **Refactor**
	- Streamlined state management in `SendDetails` and improved navigation logic in `CoinControl`.

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

File: navigation/SendDetailsStackParamList.ts [hunk 1]
```diff
@@ -3,14 +3,20 @@ import { CreateTransactionTarget, CreateTransactionUtxo, TWallet } from '../clas
 import { BitcoinUnit, Chain } from '../models/bitcoinUnits';
 
 export type SendDetailsParams = {
-  memo?: string;
+  transactionMemo?: string;
+  isTransactionReplaceable?: boolean;
+  payjoinUrl?: string;
```
[the remark this answers is not in the document]
Comment (limpbrains) [reply navigation/SendDetailsStackParamList.ts:8]: @marcosrdz seems to be correct question

[the remark this answers is not in the document]
Comment (marcosrdz) [reply navigation/SendDetailsStackParamList.ts:8]: > @marcosrdz seems to be correct question

I opted for option 1. using isTransacionReplaceable

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**